Ladybird/WebContentView.cpp

```diff
@@ -936,7 +936,7 @@ void WebContentView::notify_server_did_close_browsing_context(Badge<WebContentCl
 
 String WebContentView::notify_request_open_new_tab(Badge<WebContentClient>)
 {
-    return {};
+    return on_new_tab();
```

Comment:
We typically protect callback invocations with an `if` (see the callbacks below this one):
```c++
if (on_new_tab)
    return on_new_tab();
return {};
```